core/serial_command.py

```diff
@@ -362,30 +315,30 @@ def fire_continuous(self):
         self.disable_external_trigger()
         if self._firing is True:
             raise tellie_exception.TellieException("Cannot fire, already in firing mode")
-        self._send_command(_cmd_fire_continuous, False)
+        self._send_command(p._cmd_fire_continuous, False)
         self._firing = True
         self._force_setting = False
 
-    def read_buffer(self, n=100):
+    def read_buffer(self, n=p._read_bytes):
         return self._serial.read(n)
```

Comment:
We don't seem to use this function, but I think we should. Generally we call self._serial.read(n) directly with n set to 100. However, If we want to change the default p._read_bytes, this means we'll have to update every single instance. We should be using this method instead